### ISSUE TITLE
论文公式疑问

### ISSUE THREAD

**Issue by cyjie429** [rutgerswiselab/NS-KGE#2]
作者您好，您的工作特别好，因此拜读了您的工作，有一些疑惑，希望能得到解答，提前致谢。
`loss += self.pos_para * torch.sum(score ** 2, 0) - self.neg_para * torch.sum((3.0 - score) ** 2, 0)` 
请问这一行代码对应论文中L^P的公式，(3.0 - score)是对应预测函数吗 
为什么pos_para对应的预测函数和 neg_para 对应的预测函数代码不一样？

**Comment by lzl65825** [comment]
I appreciate your interest!
Yes, it is the predicted value for TransE.
You can take a look at Section 4.4 for more details.
Let me know if you need more help.
